packaging/windows/get-convert-licenses.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+function txt_to_rtf() {
+    INPUT="$1"
+    OUTPUT="$2"
+
+    echo "{\rtf1\ansi\deff0 {\fonttbl {\f0 Times New Roman;}}" > "$OUTPUT"
+    echo "\paperh15840 \paperw12240" >> "$OUTPUT"
+    echo "\margl720 \margr720 \margt720 \margb720" >> "$OUTPUT"
+    echo "\f0\fs24" >> "$OUTPUT"
+
+    sed s/\$/'\\line'/ "$INPUT" | sed s/\\f/'\\page'/ >> "$OUTPUT"
+    echo "}" >> "$OUTPUT"
+}
+
+if [ ! -f "gpl-3.0.txt" ]; then
+    curl -o gpl-3.0.txt "https://www.gnu.org/licenses/gpl-3.0.txt"
```

Comment:
Can we not store this in the repository itself so that we don’t need to fetch it at build time?

Reply:
> Can we not store this in the repository itself so that we don’t need to fetch it at build time?

Should we take this a step further and store the RTF as well?

Reply:
Probably, though at this point it can be added in another PR.

Reply:
Perfect, I am going to do this in the next.